src/Text/Pandoc/Readers/Org.hs

```diff
@@ -472,7 +483,7 @@ rawBlockContent (indent, blockType) = try $
   unlines . map commaEscaped <$> manyTill indentedLine blockEnder
  where
    indentedLine = try $ ("" <$ blankline) <|> (indentWith indent *> anyLine)
-   blockEnder = try $ indentWith indent *> stringAnyCase ("#+end_" <> blockType)
+   blockEnder = try $ skipSpaces *> stringAnyCase ("#+end_" <> blockType)
```

Comment:
I'm torn on whether or not I like this change.  On the plus side, this change reflects how Emacs Org-mode handles code blocks.  On the other hand, checking indentation levels allows one to write nested code blocks like this:

    #+BEGIN_SRC org
        #+BEGIN_SRC haskell
        putStrLn "Hello"
        #+END_SRC
    #+END_SRC

I think Emacs is doing it wrong here, so I'd rather not change it unless there are any good arguments to do so.